Khrystyna_Derenivska/src/app/task-item/task-item.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="list__item" [ngClass]="{'task__done': task.isDone}">
+    <div *ngIf="isEditMode; then editTask else showTask"></div>
+    <ng-template #editTask>
+        <input type="text" id="edit-task-name" [(ngModel)]='editedTaskName'>
+    </ng-template>
+    <ng-template #showTask>
+        <div class="list__item_name" [ngClass]=" ">
+            <input type="checkbox" id="list__item-1_value" class="list__item_value" (change)="onDoneChange()">
+            <label for="list__item-1_value">{{task.name}}</label>
+        </div>
+    </ng-template>
+
+
+    <div class="list__item_description">
+        <div *ngIf="isEditMode; then saveButton else editButton"></div>
+        <ng-template #editButton>
+            <button class="button button__edit" (click)="onEditClick()"
+                [attr.disabled]="task.isDone ? '' : null">Edit</button>
```

Comment:
Why do you use '' and null as value? If you need to set a disable attribute you can use only property "task.isDone" without a ternary operator.